docs/meshstack.aws.index.md

```diff
@@ -41,7 +36,7 @@ graph LR;
 
 ### IAM Roles and Service Control Policies
 
-When a user (e.g. a developer) accesses an AWS Account, they are assigned an AWS IAM role based on their project role configured on the corresponding meshProject. Operators can configure these roles and their permissions by providing an [AWS Cloud Formation](https://aws.amazon.com/cloudformation/) template. meshStack uses this template to initialize and update AWS Account configurations.
```

Comment:
note: meshIdB based IAM replication to AWS has been deprecated for a while, AWS SSO was not described in this guide before. 

I decided to not describe both variants and only describe on the AWS SSO.